src/Coordinates.java

```diff
@@ -6,4 +6,10 @@ public Coordinates (int x, int y) {
         this.x = x;
         this.y = y;
     }
+
+    //overload
```

Comment:
Wiem że to tylko komentarz ale z ciekawości czemu overload? Bo już jest constructor?
To jest normalne że jest więcej niż 1 konstruktor - tutaj masz copy constructor

Reply:
Myślałem, że overloading to jest zawsze po prostu kolejna definicja metody o tej samej nazwie